discordrb.json

```diff
@@ -57,6 +57,18 @@
             "voiceReceive": "no",
             "voiceSend": "yes",
             "voiceStateUpdate": "yes"
+        },
+        "misc": {
+            "supportsUserLogin": "yes",
+            "supportsBotLogin": "yes",
+            "cachesTokens": "yes",
+            "apiDocs": "inline",
+            "tutorials": "yes",
+            "examples": "yes",
+            "eventsAsync": "threaded",
+            "apiCallsAsync": "no",
+            "commandExtension": "yes",
+            "starsBadge": "https://img.shields.io/github/stars/meew0/discordrb.svg?style=flat-square&maxAge=2592000"
```

Comment:
A URL to a badge image that shows the current amount of stars. This will have to be handled separately on the actual page. Maybe instead of this, we could also have a customizable `badges` field which every lib can fill as they like.